src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ClientCertificates.cs

```diff
@@ -311,8 +312,28 @@ public void Manual_SendClientCertificateWithNoEKUToRemoteServer_OK()
             }
         }
 
-        private bool BackendSupportsCustomCertificateHandling =>
-            UseSocketsHttpHandler ||
-            new HttpClientHandler_ServerCertificates_Test().BackendSupportsCustomCertificateHandling;
+        private bool BackendSupportsCustomCertificateHandling
+        {
+            get
+            {
+#if TargetsWindows
+                return true;
+#else
+                if (UseSocketsHttpHandler)
```

Comment:
Do we need ifdefs like this?  Could we instead just do:
```C#
if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) || UseSocketsHttpHandler)
{
    return true;
}
```
?

Reply:
Or is the issue keeping the Interop.Http.GetSslVersionDescription call out of the Windows build?

Reply:
It's the second, we will only compile `Interop.VersionInfo.cs` for Unix build.